2200040/소득 불균형.py

```diff
@@ -0,0 +1,13 @@
+# T 테스트수 첫번째 테스트 수
+
+T = int(input())
+
+for t in range(1, T+1):
+    cnt = 0
+    a = int(input())
+    n = list(map(int, input().split()))
+    b = sum(n)/len(n)
```

Comment:
```suggestion
    a = int(input())
    n = list(map(int, input().split()))
    b = sum(n)/a
```
깔끔한 풀이군요 👍 
`a` 값을 입력을 받았으니, `len`함수를 사용하지 않고 평균을 구하는 방법도 있습니다!